client-src/elements/chromedash-roadmap-milestone-card.ts

```diff
@@ -249,6 +279,16 @@ class ChromedashRoadmapMilestoneCard extends LitElement {
           ${f.name}
         </a>
         <span class="icon_row">
+          ${this._isFeatureOutdated(f.accurate_as_of)
+            ? html`
+                <span
+                  class="tooltip"
+                  title="Feature outdated - last updated more than four weeks ago"
```

Comment:
It's not quite "updated". Perhaps:
```suggestion
                  title="Feature outdated - last checked for overall accuracy more than four weeks ago"
```